src/routers/const_router.rs

```diff
@@ -44,7 +42,7 @@ impl Router<String, Method> for ConstRouter {
             event_loop.context("Event loop must be provided to add a route to the const router")?;
 
         pyo3_asyncio::tokio::run_until_complete(event_loop, async move {
-            let output = execute_function(function, number_of_params, is_async)
+            let output = execute_http_function(&Request::default(), function)
```

Comment:
Good catch! Thanks!

Reply:
You're welcome ;)
I think it makes sense to only have one method to execute the function, as we will only have to change this one for an impact on `HttpRouter` and `ConstRouter` at the same time!